app/app/src/main/java/com/globant/calculator/android/MainActivity.java

```diff
@@ -15,6 +35,97 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-        presenter = new CalculatorPresenter(new CalculatorModel(), new CalculatorView());
+        ButterKnife.bind(this);
+        presenter = new CalculatorPresenter(new CalculatorModel(), new CalculatorView(this));
+    }
+
+    @Override
+    protected void onResume() {
+        super.onResume();
+    }
+
+    @Override
+    protected void onPause() {
+        super.onPause();
+    }
+
+    @OnClick(R.id.number_one)
+    public void oneButtonPressed() {
+        presenter.onNumberButtonPress(NUMBER_ONE);
+    }
+
+    @OnClick(R.id.number_two)
+    public void twoButtonPressed() {
+        presenter.onNumberButtonPress(NUMBER_TWO);
+    }
+
+    @OnClick(R.id.number_three)
+    public void threeButtonPressed() {
+        presenter.onNumberButtonPress(NUMBER_THREE);
+    }
+
+    @OnClick(R.id.number_four)
+    public void fourButtonPressed() {
+        presenter.onNumberButtonPress(NUMBER_FOUR);
+    }
+
+    @OnClick(R.id.number_five)
+    public void fiveButtonPressed() {
+        presenter.onNumberButtonPress(NUMBER_FIVE);
+    }
+
+    @OnClick(R.id.number_six)
+    public void sixButtonPressed() {
+        presenter.onNumberButtonPress(NUMBER_SIX);
     }
+
+    @OnClick(R.id.number_seven)
+    public void sevenButtonPressed() {
+        presenter.onNumberButtonPress(NUMBER_SEVEN);
+    }
+
+    @OnClick(R.id.number_eight)
+    public void eightButtonPressed() {
+        presenter.onNumberButtonPress(NUMBER_EIGHT);
+    }
+
+    @OnClick(R.id.number_nine)
+    public void nineButtonPressed() {
+        presenter.onNumberButtonPress(NUMBER_NINE);
+    }
+
+    @OnClick(R.id.number_zero)
+    public void zeroButtonPressed() {
+        presenter.onNumberButtonPress(NUMBER_ZERO);
+    }
+    @OnClick(R.id.add_button)
+    public void addButtonPressed() {
+        presenter.onAddButtonPressed(PLUS);
```

Comment:
use only one method in the presenter for the operators pressed

Reply:
done